spec/e2e_spec.rb

```diff
@@ -16,6 +16,8 @@
     select 'United States', from: 'billingCountry'
     fill_in 'billingPostalCode', with: '10000'
 
+    uncheck 'enableStripePass', visible: false
```

Comment:
Sometimes this checkbox is enabled by default and that requires a telephone number.